src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1364,6 +1388,32 @@ export class NetworkAreaDiagramViewer {
         }
     }
 
+    public injectDisplayCssRules(htmlElementSvg: HTMLElement) {
+        let styleTag = htmlElementSvg.querySelector('style');
+        if (!styleTag) {
+            htmlElementSvg.appendChild(document.createElement('style'));
+            console.debug('[injectDynamicCssRules] Style tag missing from SVG file. It has been created.');
+            styleTag = htmlElementSvg.querySelector('style');
+        }
+        if (styleTag && 'textContent' in styleTag) {
+            const cssSelectors = ['.nad-label-box', '.nad-text-edges', '.nad-edge-infos'];
+
+            // update the styles to display labels if needed
+            cssSelectors.forEach((selector) => {
+                const styleTags = [...document.querySelectorAll('style')].filter((styleTag) =>
+                    styleTag?.textContent?.includes(selector)
+                );
+                styleTags.forEach((styleTag) => {
+                    styleTag.textContent =
+                        styleTag?.textContent !== null
+                            ? styleTag?.textContent.replace(`{display: none;}`, `{display: block;}`)
```

Comment:
This line bothers me : there is an unnecessary "?", and if the css rule is slightly different, it will not work.